dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/scheduling/DefaultJobConfigurationService.java

```diff
@@ -136,6 +137,15 @@ public List<JobConfiguration> getAllJobConfigurations()
         return jobConfigurationStore.getAll();
     }
 
+    @Override
+    @Transactional( readOnly = true )
+    public List<JobConfiguration> getJobConfigurations( JobType type )
```

Comment:
There is no dedicated store interface/class yet and this seems too little to start creating one so we could do this as a DB filter so I decided I do it in-memory for now.